<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimeSetWidget</class>
 <widget class="QWidget" name="AnimeSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic Animation Set</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="animVList" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QListWidget" name="animList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="animLButtons">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="animLNewButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="animLClearButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="animLCountButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Count</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>450</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>800</width>
       <height>400</height>
      </size>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>42</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Block Offset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Flying Offset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Invincibility Offset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="flyingOffSpinBox">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="invincibleOffSpinBox">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="blockOffSpinBox">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>152</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Animation Pattern</string>
      </property>
      <widget class="QPushButton" name="animPatButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
      <widget class="QListWidget" name="animPatList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>181</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>2</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>352</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sample</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sampleComboBox">
      <property name="geometry">
       <rect>
        <x>57</x>
        <y>352</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sampleCountSpinBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>352</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>352</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>of</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sampleListButton">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>350</y>
        <width>31</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>List</string>
      </property>
     </widget>
     <widget class="QFrame" name="previewFrame">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>352</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>-340</x>
         <y>-430</y>
         <width>400</width>
         <height>462</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>800</width>
         <height>400</height>
        </size>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>201</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Block Offset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Flying Offset</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Invincibility Offset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="flyingOffSpinBox_2">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="invincibleOffSpinBox_2">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="blockOffSpinBox_2">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>201</width>
          <height>191</height>
         </rect>
        </property>
        <property name="title">
         <string>Animation Pattern</string>
        </property>
        <widget class="QPushButton" name="animPatButton_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>75</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
        <widget class="QListWidget" name="animPatList_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>181</width>
           <height>131</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>370</y>
          <width>49</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sample</string>
        </property>
       </widget>
       <widget class="QComboBox" name="sampleComboBox_2">
        <property name="geometry">
         <rect>
          <x>67</x>
          <y>370</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sampleCountSpinBox_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>370</y>
          <width>40</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>370</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>of</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sampleListButton_2">
        <property name="geometry">
         <rect>
          <x>235</x>
          <y>368</y>
          <width>31</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>List</string>
        </property>
       </widget>
       <widget class="QFrame" name="previewFrame_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>370</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>395</y>
          <width>116</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Frame (Start Position)</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="frameStartSpinBox_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>395</y>
          <width>40</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>377</y>
        <width>116</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Frame (Start Position)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="frameStartSpinBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>377</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
